### DISCUSSION TITLE
Using K3s

### DISCUSSION THREAD

**Discussion by sivanandasubbiah** [k3s-io/k3s#4908] [Q&A]
Hi,

We intend to use K3s for one of our products, I have the following technical queries related to the K3s:
Will it be possible to host and run K3s on top of RHEL platform, If so do K3s have any dependencies with RHEL RPMs. Or is there any problem on the co-existence of K3s with any RHEL RPMs.
What is the basic or minimum system resource / requirements for running K3s.
Can you clarify with details or point to me to the correct document where I can get these info.

Regards.
Siva.

**Comment by luisbrandao** [comment]
I been using k3s exclusively on my centos-7 cluster (baremetal) without many issues.
I am using docker instead containerd, for this to work i had to change the docker.service to use cgroupfs.
didn't any hard roadblock so far

I did even prototype a centos-7 cluster on AWS with HA on master + RDS postgresql.
